docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -3,85 +3,205 @@ title: teleport-cluster Chart Reference
+used as the cluster's public address by default.
 
 <Admonition type="note">
-  If using a fully qualified domain name as your `clusterName`, you will also need to configure the DNS provider for this domain to point
-  to the external load balancer address of your Teleport cluster.
+When using a fully qualified domain name as your `clusterName`, you will also
+need to configure the DNS provider for this domain to point to the external
+load balancer address of your Teleport cluster.
 
-  (!docs/pages/kubernetes-access/helm/includes/kubernetes-externaladdress.mdx!)
+(!docs/pages/kubernetes-access/helm/includes/kubernetes-externaladdress.mdx!)
 
-  You will need to manually add a DNS A record pointing `teleport.example.com` to either the IP or hostname of the Kubernetes load balancer.
+You will need to manually add a DNS A record pointing `teleport.example.com` to
+the IP, or a CNAME record pointing to the hostname of the Kubernetes load balancer.
 
 <Details title="Using Application Access?">
 (!docs/pages/includes/dns-app-access.mdx!)
 </Details>
-
-  If you are not using ACME certificates, you may also need to accept insecure warnings in your browser to view the page successfully.
 </Admonition>
 
+<Admonition type="warning">
+The `clusterName` cannot be changed during a Teleport cluster's lifespan.
+If you need to change it, you must redeploy a completely new cluster.
+</Admonition>
 
 ## `kubeClusterName`
 
-| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
-| - | - | - | - | - |
-| `string` | `clusterName` value | no | `kubernetes_service.kube_cluster_name` |  ✅ |
+| Type | Default value | Required? | `teleport.yaml` equivalent |
+| - | - | - | - |
+| `string` | `clusterName` value | no | `kubernetes_service.kube_cluster_name` |
+
+`kubeClusterName` sets the name used for Kubernetes access.
+This name will be shown to Teleport users connecting to the Kubernetes cluster.
+
+## `auth`
+| Type     | Default value | Required? |
+|----------|---------------|-----------|
+| `object` | {}            | no        |
+
+The `teleport-cluster` chart deploys two sets of pods: auth and proxy.
+`auth` contains values specific for the auth pods. You can use it to
+set specific values for auth pods, taking precedence over chart-scoped values.
+
+For example, to override the [`postStart`](#postStart) value only for auth pods:
+```yaml
+# By default all pods postStart command should be "echo starting"
+postStart: ["echo", "starting"]
+
+auth:
+  # But we override the `postStart` value specifically for auth pods
+  postStart: ["curl", "http://hook"]
+  imagePullPolicy: Always
+```
+
+### `auth.teleportConfig`
+
+| Type     | Default value | Required? |
+|----------|---------------|-----------|
+| `object` | {}            | no        |
+
+`auth.teleportConfig` contains YAML teleport configuration for auth pods.
+The configuration will be merged with the chart-generated configuration
```

Comment:
Is it worth it to clarify what the merge rules are? For example, if the field is a list, do these get merged or override?

Reply:
Yes it is, because I'm not 100% sure of the answer and it is not documented in sprig. I would expect the list to be overridden. I'll do some tests